### PR TITLE
Rebase aptogotchi with personal & Add ANS feature

### PR DESCRIPTION
**Pull request by darren-wangg**
- fast forward this aptogotchi version with the [live](https://aptogotchi.vercel.app/) version (which was a personal fork)

- add ANS package as "Owner" field

Screenshot 2023-10-11 at 3 17 51 PM
Screenshot 2023-10-11 at 3 19 50 PM
Screenshot 2023-10-11 at 3 20 17 PM

- add comments to Contract

### CODE REVIEW

**Comment by angieyth** [comment]
can you separate the rebase and the ans change into 2 PRs? thank you!!

File: frontend/src/app/layout.tsx [hunk 1]
```diff
@@ -20,6 +21,7 @@ export default function RootLayout({ children }: PropsWithChildren) {
     <html lang="en">
       <body className={kongtext.className}>
         <WalletProvider>{children}</WalletProvider>
+        <Analytics />
```
Comment (angieyth) [inline frontend/src/app/layout.tsx:24]: what does this for?

File: frontend/src/components/WalletButtons/index.tsx [hunk 2]
```diff
@@ -23,10 +23,10 @@ export const WalletButtons = () => {
           className="bg-gray-50 text-sm rounded-lg block p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white"
           style={{ marginRight: "15px" }}
         >
-          <option selected value="Devnet">
-            Devnet
+          <option value="Devnet">Devnet</option>
+          <option selected value="Testnet">
+            Testnet
```
Comment (angieyth) [inline frontend/src/components/WalletButtons/index.tsx:28]: do we actually support different network? or this is just a placeholder

File: move/sources/aptogotchi.move [hunk 3]
```diff
@@ -265,4 +259,55 @@ module aptogotchi::main {
...
+        let name = string::utf8(b"Aptogotchi");
+        let parts = vector[0, 0, 0];
+        create_aptogotchi(&caller, name, parts);
+
+        // verify default struct values
+        assert!(get_name(&caller) == name, 1);
+        assert!(get_health_points(&caller) == HP_UPPER_BOUND, 1);
+        assert!(get_happiness(&caller) == HAPPINESS_UPPER_BOUND, 1);
+        assert!(get_parts(&caller) == parts, );
+    }
```
Comment (angieyth) [inline move/sources/aptogotchi.move:312]: can you remove those?

**Review by 0xZihan — APPROVED** [review]

File: move/sources/aptogotchi.move (hunk 3 above)
Comment (darren-wangg) [reply move/sources/aptogotchi.move]: done

File: frontend/src/components/WalletButtons/index.tsx (hunk 2 above)
Comment (darren-wangg) [reply frontend/src/components/WalletButtons/index.tsx]: removed it

File: frontend/src/app/layout.tsx (hunk 1 above)
Comment (darren-wangg) [reply frontend/src/app/layout.tsx]: will remove for now (this is to track Visitors once we deploy this repo)

**Pull request merged**